cognite/neat/rules/importers/_rdf/_base.py

```diff
@@ -104,7 +105,40 @@ def to_rules(
         rules_dict = self._to_rules_components()
 
         rules = InformationInputRules.load(rules_dict)
+
+        self._add_transformation(rules)
+
         return ReadRules(rules, self.issue_list, {})
 
     def _to_rules_components(self) -> dict:
         raise NotImplementedError()
+
```

Comment:
this is now more explicit and easier way to build up both prefixes as well as transformations